tests/Converter/Validator/Xsd2ValidatorBase.php

```diff
@@ -0,0 +1,194 @@
+<?php
+namespace GoetasWebservices\Xsd\XsdToPhp\Tests\Converter\Validator;
+
+use GoetasWebservices\Xsd\XsdToPhp\Jms\YamlValidatorConverter;
+use GoetasWebservices\Xsd\XsdToPhp\Naming\ShortNamingStrategy;
+use GoetasWebservices\XML\XSDReader\SchemaReader;
+
+abstract class Xsd2ValidatorBase extends \PHPUnit_Framework_TestCase
```

Comment:
currently there is only one class that inherits from this (`Xsd2ValidatorTest`) isn't this premature architecture?